### PR TITLE
Re-implement a wrench that was locked solely to aftershock

### PR DESCRIPTION
**Pull request by HotCupOfJava**
SUMMARY: Content "Re-implement the chain wrench"

#### Purpose of change

The chain wrench is a real life tool that is far easier craft out of scrap than a normal wrench. It can be used for bolt turning nearly as effective as a normal wrench though.

#### Describe the solution

Add a recipe and the item file to the main game locations

#### Describe alternatives you've considered

Make wrenches more common

#### Testing

I spawned in a character and tested if I had access to the recipe

### CODE REVIEW

**Comment by John-Candlebury** [comment]
You should probably remove it from aftershock too!

File: data/json/recipes/other/tool.json [hunk 1]
```diff
@@ -1122,5 +1122,19 @@
...
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "1 h",
+    "reversible": true,
+    "autolearn": true,
+    "skills_required": [ "mechanics", 2 ],
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "SAW_M", "level": 2 }, { "id": "DRILL", "level": 3 } ],
+    "components": [ [ [ "scrap", 20 ] ], [ [ "pipe", 1 ] ], [ [ "chain", 1 ] ] ]
```
Comment (anothersimulacrum) [inline data/json/recipes/other/tool.json:1138]: This recipes does not produce any of these:
image
image
image
image

**Comment by ZhilkinSerg** [comment]
> 
> 
> Shouldn't the recipe ask for a chain?

And a specific one probably.

**Review by kevingranade — CHANGES_REQUESTED** [review]
This needs a bicycle chain as an input of the recipe.
It also needs a more robust set of requirements for the handle assembly, there's a reason these things are usually beefy forged things, though there are various sources that indicate you can get away with welding on a pipe.

**Comment by ZhilkinSerg** [comment]
Feel free implement requested changes and reopen.

**Pull request closed without merging**